containers/docker/tools/lsb_compat.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# This removes /etc/init.d service if service exists in runit.
+# It also creates a symlink from /usr/bin/sv to /etc/init.d/$service
+# to support SysV syntax: service $service <command> or /etc/init.d/$service <command>
```

Comment:
True :) It can avoid many potential issues with services management, even with tools that we don't really control in the project.